Buildings/Experimental/OpenBuildingControl/CDL/Logical/Composite/OnHold.mo

```diff
@@ -0,0 +1,102 @@
+within Buildings.Experimental.OpenBuildingControl.CDL.Logical.Composite;
+model OnHold "Block that holds a signal on for a requested time period"
+
+  LessThreshold                                               les1(threshold=
+        holdOnDuration)
+    annotation (Placement(transformation(extent={{-20,-40},{0,-20}})));
```

Comment:
Please correct indentation and add comments to each instance.
Also, this assignment should probably be `final` as there is no use to ever overwrite it.